packages/react-native-session-replay/android/src/test/kotlin/com/datadog/reactnative/sessionreplay/DdSessionReplayImplementationTest.kt

```diff
@@ -74,6 +75,7 @@ internal class DdSessionReplayImplementationTest {
     ) {
         // Given
         val sessionReplayConfigCaptor = argumentCaptor<SessionReplayConfiguration>()
+        val coreCaptor = argumentCaptor<SdkCore>()
```

Comment:
why it is needed, if we don't use captured value? instead we can use `any()` there, this will be more correct logically. Same for another test below.